app/app/guide/page.tsx

```
@@ -0,0 +1,280 @@
+
+const sectionHeader = "mb-2 text-[10px] font-medium uppercase tracking-[0.25em] text-[var(--accent)]/60";
+const sectionTitle = "text-lg font-semibold tracking-[-0.01em] text-white sm:text-xl";
+const cardClass = "border border-[var(--border)] bg-[var(--panel-bg)]";
+const cellClass = "px-4 py-3 text-[12px]";
+const thClass = "px-4 py-3 text-[10px] font-medium uppercase tracking-[0.15em] text-[var(--text-muted)] text-left";
+
+function Section({ tag, title, children }: { tag: string; title: string; children: React.ReactNode }) {
+  return (
+    <section className="space-y-4">
+      <div>
+        <div className={sectionHeader}>// {tag}</div>
```

Biome (2.3.13)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)